int32 digit_1
int32 digit_2
int32 digit_3
---
string status